### PR TITLE
Bunch o' terragov stuff.

### PR DESCRIPTION
**Pull request by CRITAWAKETS**
## About The Pull Request

Terragov has upgraded since their first appearance. Now they have a Field Operating Base, on the centcom z-level. They do have to use the centcom ferry though, as to not get shot down by Space Station 13's mysteriously only now said defenses. (it totally isnt just me being too lazy to add an entire new shuttle dock to every single map)

In addition, TerraGov Elites now have new suit sprites, courtesy of AzlanonPC.

Aswell, TerraGov Elites now have energy swords. They use a plasma sword design which is totally not stolen from a covenant of alien races. The design is more damaging and more armour piercing than a regular e-sword, having as much damage as a desword for only one hand, however it cannot block projectiles. There is a CTF-only version that has been added that is an instant-kill (however because of how CTF works, you need to down their energy shields)

The TerraGov AR and its ammunition have also recieved changes to their sprites.

## Why It's Good For The Game

I dont think anybody can argue against improvements. Also, TerraGov elites are currently massively underpowered for what they are supposed to be so giving them an energy sword feels deserved.

## Changelog
:cl:
add: TerraGov has upped their arsenal (and budget). They now have a base of operations, energy swords, new armor for their elites and a new AR design.
/:cl:

It's energy sword sunday.

### CODE REVIEW

**Comment by dee-dubya** [comment]
Currently the biggest issue I've had with them is that the only way for admins to spawn them in currently is via right click>set equipment, which isn't ideal for getting ghosts to take it up.

Considering you're giving them their own spot on the admin z-level now, it'd be appreciated if you could code them in as an actual centcom response team option for ease of use on our end.

**Comment by dinnerwoerror** [comment]
im sure you could just steal and repurpose the code for ERTs/ deathsquads

File: waspstation/code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -14,3 +14,15 @@
...
+	name = "TerraGov Briefing Room"
+
+/area/centcom/terragov/equipment
+	name = "TerraGov Equipment Room"
+
+/area/centcom/terragov/barracks
+	name = "TerraGov Barracks"
+
+/area/centcom/terragov/bay
+	name = "TerraGov Shuttle Bay"
```
Comment (Superyodeler) [inline waspstation/code/game/area/Space_Station_13_areas.dm:28]: Add a newline here
```suggestion
	name = "TerraGov Shuttle Bay"

```

File: waspstation/code/game/objects/items/melee/energy.dm [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	if(. && active)
+		icon_state = "plasmasword1"
+
+/obj/item/melee/transforming/energy/ctf/terragov
+	armour_penetration = 40
+	force_on = 34 //desword grade, but 0 blocking
+
+/obj/item/melee/transforming/energy/ctf/transform_messages(mob/living/user, supress_message_text)
+	playsound(user, active ? 'waspstation/sound/weapons/terragov_sword_arm.ogg' : 'sound/weapons/saberoff.ogg', 35, TRUE)
+	to_chat(user, "<span class='notice'>[src] [active ? "is now active":"can now be concealed"].</span>")
```
Comment (Superyodeler) [inline waspstation/code/game/objects/items/melee/energy.dm:30]: And here
```suggestion
	to_chat(user, "<span class='notice'>[src] [active ? "is now active":"can now be concealed"].</span>")

```

**Pull request merged**